src/libltfs/ltfs.h

```diff
@@ -698,6 +707,11 @@ int ltfs_print_device_list(struct tape_ops *ops);
 void ltfs_enable_livelink_mode(struct ltfs_volume *vol);
 
 int ltfs_profiler_set(uint64_t source, struct ltfs_volume *vol);
+
+int ltfs_get_rao_list(char *path, struct ltfs_volume *vol);
+int _ltfs_write_rao_file(char *write_data, char *file_path, size_t *write_size);
+int _ltfs_read_rao_file(char *read_data, uint32_t *num_of_files, char *file_path);
```

Comment:
Same as `_ltfs_write_rao_file()`

Reply:
modified in https://github.com/LinearTapeFileSystem/ltfs/pull/288/commits/87b5a2d44974149001f8f29ae12a3718b102e69f